stix_shifter_modules/alertflex/stix_transmission/alertflex_connector.py

```diff
@@ -0,0 +1,72 @@
+from stix_shifter_utils.modules.base.stix_transmission.base_sync_connector import BaseSyncConnector
+from stix_shifter_utils.utils.error_response import ErrorResponder
+from .api_client import APIClient
+import json
+
+
+class UnexpectedResponseException(Exception):
+    pass
+
+
+class Connector(BaseSyncConnector):
+    def __init__(self, connection, configuration):
+        self.api_client = APIClient(connection, configuration)
+
+    def _handle_errors(self, response, return_obj):
+        response_code = response.code
+        response_txt = response.read().decode('utf-8')
+
+        if 200 <= response_code < 300:
+            return_obj['success'] = True
+            if response_txt:
+                response_json = json.loads(response_txt)
+                if 'results' in response_json:
+                    return_obj['data'] = response_json['results']
+        elif ErrorResponder.is_plain_string(response_txt):
+            ErrorResponder.fill_error(return_obj, message=response_txt)
+        elif ErrorResponder.is_json_string(response_txt):
+            response_json = json.loads(response_txt)
+            ErrorResponder.fill_error(return_obj, response_json, ['reason'])
+        else:
+            raise UnexpectedResponseException
+        return return_obj
+
+    def ping_connection(self):
+        response_txt = None
+        return_obj = {}
+        try:
+            response = self.api_client.ping_box()
+            return self._handle_errors(response, return_obj)
+        except Exception as e:
+            if response_txt is not None:
+                ErrorResponder.fill_error(return_obj, message='unexpected exception')
+                print('can not parse response: ' + str(response_txt))
```

Comment:
we have recently introduced python logging into stix-shifter to move away from standard output using print. you should update your repo and make use of that. here's an example: https://github.com/opencybersecurityalliance/stix-shifter/blob/77ea7c36a6c46b951db89fa8dc62278457f814e6/stix_shifter_modules/bigfix/stix_transmission/ping_connector.py#L35